test/integration_test_runner.rb

```diff
@@ -1,6 +1,10 @@
 require 'test/unit'
 require_relative '../lib/pmdtester/runner'
 class IntegrationTestRunner < Test::Unit::TestCase
+  def setup
+    `rake clean`
+  end
+
   def test_local_mode
     Process.fork do
```

Comment:
I think, we should change the integration tests, to execute the shell script `bin/pmdtester`, so that we have this covered in the tests, too.

You might need to call it again via bundler, e.g. `bundle exec bin/pmdtester -r target/repositories/pmd ...`.